### DISCUSSION TITLE
Errore with upload on team drive.

### DISCUSSION THREAD

**Discussion by jenny2000000000** [anasty17/mirror-leech-telegram-bot#1418] [Q&A]
I have this errore after download the torrent "EOF occurred in violation of protocol (_ssl.c:2396)". There Is a solution for this?

**Comment by anasty17** [comment]
Use vpn for docker. This what other doing. This not related to my code. Search in group.

**Reply by jenny2000000000** [reply]
I'm sorry but I did not understand.  after downloading a torrent, the bot fails to upload to the drive, giving me that error.  I also tried to format the vps, update it but nothing changes.

**Reply by anasty17** [reply]
https://t.me/mltb_official_group/11118

**Comment by annibal1** [comment]
I also have a mistake on the bot
[Erno 32] Broken Pipe

**Reply by anasty17** [reply]
https://t.me/mltb_official_group/11118

**Comment by anasty17** [comment]
Not my code issue. This from vps provider. Idk what the issue exactly.
